std/array.d

```diff
@@ -2840,18 +2840,18 @@ if (isDynamicArray!A)
         }
     }
 
-  // emplace an item onto an array.
-  void emplacePut(Args...)(Args args) {
-	import std.conv: emplaceRef;
+	// emplace an item onto an array.
+	void emplacePut(Args...)(Args args) {
+		import std.conv: emplaceRef;
 
-	ensureAddable(1);
-	immutable len = _data.arr.length;
+		ensureAddable(1);
+		immutable len = _data.arr.length;
 
-	auto bigData = (() @trusted => _data.arr.ptr[0..len + 1])();
-	emplaceRef!(Unqual!T)(bigData[len], args);
-	// don't set this back until we're sure we have the item initialized.
-	_data.arr = bigData;
-  }
+		auto bigData = (() @trusted => _data.arr.ptr[0..len + 1])();
+		emplaceRef!(Unqual!T)(bigData[len], args);
+		// don't set this back until we're sure we have the item initialized.
+		_data.arr = bigData;
+	}
```

Comment:
If you click on the "autotester" you see that it is failing due to that.
In doubt, just manually run `rdmd ../dmd/src/checkwhitespace.d std/array.d`